grants_tagger/streamlit_visualize.py

```diff
@@ -0,0 +1,48 @@
+import streamlit as st
+import pandas as pd
+
+from grants_tagger import predict_tags
+
+threshold = st.sidebar.slider("Threshold", min_value=0.0, max_value=1.0, value=0.5)
+text = st.text_area('Grant abstract', 'The cell is...', height=300)
+
+models = {
+    "disease_mesh_cnn-2021.03.1": {
```

Comment:
Nice model dispatcher, although in the future you can probably accept the model config file as parameter to your `grants_tagger visualize`.